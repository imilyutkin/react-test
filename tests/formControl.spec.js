import React from 'react';
import {shallow, sinon} from 'enzyme';
import FormControl from '../src/components/shared/formControl';
import renderer from 'react-test-renderer';


test('Form control render testing', () => {
        const formControl = renderer.create(
            <FormControl />
        );

        let tree = formControl.toJSON();
        expect(tree).toMatchSnapshot();
    }
);

test('Form control render inside', () => {
        const formControl = shallow(
            <FormControl>
                <label>Hello</label>
            </FormControl>
        );

        expect(formControl.find('label').text()).toBe("Hello");
    }
);


test('Form control submit testing', () => {
        const formControl = renderer.create(
            <FormControl />
        );
    }
);