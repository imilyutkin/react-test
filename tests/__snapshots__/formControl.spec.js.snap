// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form control render testing 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    type="submit"
  >
    Save
  </input>
</form>
`;
